validator/valnode/valnode.go

```diff
@@ -0,0 +1,126 @@
+package valnode
+
+import (
+	"context"
+
+	"github.com/offchainlabs/nitro/validator"
+
+	"github.com/ethereum/go-ethereum/node"
+	"github.com/ethereum/go-ethereum/rpc"
+	flag "github.com/spf13/pflag"
+
+	"github.com/offchainlabs/nitro/validator/server_api"
+	"github.com/offchainlabs/nitro/validator/server_arb"
+	"github.com/offchainlabs/nitro/validator/server_common"
+	"github.com/offchainlabs/nitro/validator/server_jit"
+)
+
+type WasmConfig struct {
+	RootPath string `koanf:"root-path"`
+}
+
+func WasmConfigAddOptions(prefix string, f *flag.FlagSet) {
+	f.String(prefix+".root-path", DefaultWasmConfig.RootPath, "path to machine folders, each containing wasm files (replay.wasm, wasi_stub.wasm, soft-float.wasm, go_stub.wasm, host_io.wasm, brotli.wasm")
```

Comment:
Unrelated to this PR but we should probably say `path to machine folders, each containing wasm files (machine.wavm.br, replay.wasm)` because we aren't shipping the other wasm libraries with our docker images any more.